docs/v3/core/v3.0.rst

```diff
@@ -1601,6 +1590,9 @@ a prefix will erase all the implicit group in the prefix.
 Care must thus be taken when erasing an array or a group if the parent needs to
 be converted into an explicit group.
 
+A race-condition arises if a client writes an array at path ``P``,
+and another concurrently assumes ``P`` is an implicit group and writes subgroups or arrays into it.
```

Comment:
```suggestion
and another concurrently assumes ``P`` is an implicit group and writes subgroups or arrays into it.
Implementations may choose to never use implicit groups to avoid this.
```